lib/schema.coffee

```diff
@@ -47,11 +47,11 @@ module.exports =
 			description: 'user id'
 			type: 'number'
 			minimum: 0
-			required: true
+			required: false
 		username:
 			description: 'username'
 			type: 'string'
-			required: true
+			required: false
```

Comment:
I think it makes sense to remove it from the schema completely if it's ignored.

Reply:
@dfunckt I think removing it would cause problems if you try to validate a config that includes them. Honestly I think the right move would be to deprecate/remove this module but for now this gets what we need

Reply:
Sounds good @Page-